app/controllers/miq_request_controller.rb

```diff
@@ -585,22 +588,17 @@ def request_edit_settings(provision_request)
   end
 
   def get_session_data
+    super
     @title        = _("Requests")
     @request_tab  = session[:request_tab] if session[:request_tab]
-    @layout       = layout_from_tab_name(@request_tab)
```

Comment:
I don't think you can drop this line, the mixin will default to prefix, but won't do the logic that's being done in `layout_from_tab_name`